arches_lingo/src/arches_lingo/components/scheme/editor/SchemeEditor.vue

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+const MAXIMIZE = "maximize";
+const SIDE = "side";
+const CLOSE = "close";
+</script>
+
+<script setup lang="ts">
+import { useGettext } from "vue3-gettext";
+import Button from "primevue/button";
+
+const { $gettext } = useGettext();
+
+const props = defineProps<{
+    editorMax: boolean;
+}>();
+
+const emit = defineEmits([MAXIMIZE, SIDE, CLOSE]);
+const toggleSize = () => {
+    if (props.editorMax) {
+        emit(MAXIMIZE);
+    } else {
+        emit(SIDE);
+    }
+};
+</script>
+
+<template>
+    <div class="header">
+        <div>
+            <h2>{{ $gettext("Scheme Editor Tools") }}</h2>
+            <div>
+                <Button @click="toggleSize"
```

Comment:
nit: probably the formatter's fault, but IMO this should be either

```
<Button @click="toggleSize">
```

or 

```
<Button 
    @click="toggleSize"
>
```

Reply:
Not sure why the formatter originally allowed this to happen - but it did!  Maybe there were more attributes on this button originally.